app/src/main/java/com/beemdevelopment/aegis/vault/Vault.java

```diff
@@ -16,6 +16,9 @@ public class Vault {
     private final UUIDMap<VaultEntry> _entries = new UUIDMap<>();
     private final UUIDMap<VaultGroup> _groups = new UUIDMap<>();
 
+    // Whether we've migrated the group list to the new format while parsing the vault
+    private boolean _groupsMigrated = false;
```

Comment:
Could you please change this to _areGroupsMigrated, _groupsNeedsMigration or something similiar?

Reply:
I've settled on ``isGroupsMigrationFresh``.